<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportarCSVDialog</class>
 <widget class="QDialog" name="ExportarCSVDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exportar a CSV</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>¿Qué desea Exportar?</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rdProcesos">
            <property name="text">
             <string>Procesos</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdDemandantes">
            <property name="text">
             <string>Clientes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdDemandados">
            <property name="text">
             <string>Contrapartes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdJuzgados">
            <property name="text">
             <string>Juzgados</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdActuaciones">
            <property name="text">
             <string>Actuaciones</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdActuacionesCriticas">
            <property name="text">
             <string>Eventos Próximos</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Formato</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rdCSV">
            <property name="text">
             <string>Valores Separados por Comas</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdTab">
            <property name="text">
             <string>Valores Separados por Tabulaciones</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportarCSVDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportarCSVDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
